include/openmapper_desktop/window.h

```diff
@@ -25,25 +23,26 @@ class Window : public QWidget {
   Q_OBJECT
 
  public:
-  explicit Window(QWidget *parent = 0);
+  explicit Window(QWidget* parent = 0);
   ~Window();
 
  protected:
-  void keyPressEvent(QKeyEvent *event);
-  void timerEvent(QTimerEvent *event);
+  void keyPressEvent(QKeyEvent* event);
+  void timerEvent(QTimerEvent* event);
   void initialize_input();
 
  private:
-  Ui::Window *ui;
+  Ui::Window* ui;

```

Comment:
Dont know right now why all pointer signs are left instead of right. Will correct that later